lib/algos/list/tools.js

```diff
@@ -1,3 +1,6 @@
+const VSConst = require('../../versioning/constants').VersioningConstants;
```

Comment:
Could combine these two lines into one, I didn't see VSConst referenced anywhere.

Reply:
Yes it's just to fetch the DbPrefixes. I copied this pattern from other files that do this, but indeed I could make it a single line.